src/modelbench/benchmark_runner.py

```diff
@@ -185,6 +109,10 @@ def __init__(self, runner: "TestRunnerBase"):
         self.run_tracker = runner.run_tracker
         self.completed_item_count = 0
 
+        self.data_dir.mkdir(exist_ok=True, parents=True)
+        self.run_id = datetime.now().strftime("run-%Y%m%d-%H%M%S-%f")
+        self.journal = RunJournal(self.data_dir / f"journal-{self.run_id}.jsonl")
```

Comment:
The data directory is used to stored data needed for a run e.g. dependencies, caches. Would it be possible to store the journal in a separate directory? Modelgauge writes its records to "output" by default-- the journals could go there as well.

Reply:
Well right now this is storing to a directory called `run`, and this seems like run stuff to me. I see the merit in splitting up input and and output, but caches are kinda in between. How about when folks are back we talk it over and figure out something that seems sensible for what we have and has room for what we expect in the future?

Reply:
Could we at least store them in a subdirectory like `run/journals` in the meantime? To keep the `run` directory organized.

Reply:
Great idea.